Source/IBMCloudAppID/internal/Config.swift

```diff
@@ -16,12 +16,12 @@ import BMSCore
 
 internal class Config {
     
-    private static var oauthEndpoint = "/oauth/v3/"
+    private static var oauthEndpoint = "/oauth/v4/"
     private static var attributesEndpoint = "/api/v1/"
     private static var serverUrlPrefix = "https://appid-oauth"
     private static var attributesUrlPrefix = "https://appid-profiles"
     private static var publicKeysEndpoint = "/publickeys"
-    private static var urlProtocol = "http"
+    private static var urlProtocol = "https"
```

Comment:
this is always true since it always points to App ID and app id is always on https.